src/doc/data_into_visit/data_examples/xycurve.xmf

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<!DOCTYPE Xdmf SYSTEM "Xdmf.dtd" []>
+<Xdmf xmlns:xi="http://www.w3.org/2001/XInclude" Version="2.0">
+    <Domain>
+        <Grid Name="Curve" GridType="Uniform" GridPurpose="Curve">
```

Comment:
@biagas and @brugger1...I am adding support for 1D curves to Xdmf plugin. But, Xdmf data model doesn't really support one dimensional objects. So, I am defining a special case for a `2DRectMesh` object which has more than 1 point in X dimensions and only one point in Y dimension.

But, to really lock it down, I am also defining a new XML tag, `GridPurpose` with value `"Curve"` which must be present too as you can see in the `<Grid...` line above. Now, that is an XML tag. In other words, its XML metadata that appears *inside* the Xdmf `<Grid>` element. It is harmless to Xdmf. Xdmf lib and consumers will read it (it gets gobbled up with all the XML) but none but the VisIt plugin will attempt to look for it.

Thoughts?

Reply:
Sounds good to me.